### PR TITLE
Implement tests for Actions.add/2

### PR DESCRIPTION
**Pull request by wevtimoteo**
- [x] Add mox package to help us defining mocks
- [x] Introduce command runner to abstract `System`
- [x] Implement tests for `Actions.add/2`

### CODE REVIEW

File: lib/git_pair/system_behaviour.ex [hunk 1]
```diff
@@ -0,0 +1,5 @@
+defmodule GitPair.SystemBehaviour do
+  @moduledoc false
+
+  @callback cmd(String.t(), list()) :: { String.t(), integer }
```
[the remark this answers is not in the document]
Comment (wevtimoteo) [reply lib/git_pair/system_behaviour.ex]: Done in https://github.com/elixirsc/git-pair/pull/23/commits/739e9d7c03656c475dc8a5d3e7e6e573d12c2287

File: test/actions_test.exs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+defmodule GitPair.ActionsTest do
+  use ExUnit.Case, async: true
+  import Mox
+  alias GitPair.Actions
+
+  setup :verify_on_exit!
+
+  test ".add calls git config add comma" do
+    GitPair.SystemMock
+    |> expect(:cmd, fn _cmd, _options -> {"", 0} end)
```
[the remark this answers is not in the document]
Comment (wevtimoteo) [reply test/actions_test.exs]: Done in https://github.com/elixirsc/git-pair/pull/23/commits/739e9d7c03656c475dc8a5d3e7e6e573d12c2287

File: test/actions_test.exs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    { result, message } = Actions.add(["fake-user"])
```
[the remark this answers is not in the document]
Comment (wevtimoteo) [reply test/actions_test.exs]: Done in https://github.com/elixirsc/git-pair/pull/23/commits/739e9d7c03656c475dc8a5d3e7e6e573d12c2287

File: lib/git_pair/actions.ex [hunk 4]
```diff
@@ -39,14 +39,16 @@ defmodule GitPair.Actions do
...
   defp command(action, [username]) do
-    System.cmd("git", [@git_config, action, @key, username])
+    command_runner().cmd("git", [@git_config, action, @key, username])
   end
 
   defp command(_action, _usernames) do
     {:error, "Unsuported multiple users"}
   end
+
+  def command_runner(), do: Application.get_env(:git_pair, :command_runner)
```
Comment (leandroh) [inline lib/git_pair/actions.ex:53]: 👏

File: test/actions_test.exs [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  use ExUnit.Case, async: true
+
+  import Mox
+
+  alias GitPair.Actions
+  alias GitPair.SystemMock
+
+  setup :verify_on_exit!
+
+  test ".add calls git config add comma" do
```
Comment (leandroh) [inline test/actions_test.exs]: What this test means?

**Review by leandroh — APPROVED** [review]
Consider my comments then 🚀

File: test/actions_test.exs (hunk 5 above)
Comment (wevtimoteo) [reply test/actions_test.exs]: **praise:** nice catch! :baseball: 

Will fix tonight

Comment (wevtimoteo) [reply test/actions_test.exs]: Done in a6d0f21

**Pull request merged**